docker-compose.yml

```diff
@@ -110,6 +137,8 @@ services:
     depends_on:
       - db
       - groot-api-gateway
+    volumes:
+      - ./groot-users-service:/usr/src/app
```

Comment:
Do we need to add `RACK_ENV=development` to the ruby services? Or is development assumed by default?

Reply:
Development is the default